packages/app/src/app.ts

```diff
@@ -1,6 +1,6 @@
 import createClient from "openapi-fetch";
 
-import { paths } from "./schema";
+import { paths, components} from "./schema";
```

Comment:
sorry for being pick, we still have to include the prettier check in the CI.
There is a missing space here.

Reply:
No worries 😅 😆